contracts/callpaths/ColdPath.sol

```diff
@@ -131,7 +131,8 @@ contract ColdPath is MarketSequencer, DepositDesk, ProtocolAccount {
         (int128 baseFlow, int128 quoteFlow) = initCurve(pool, price, initLiq);
         settleInitFlow(lockHolder_, base, baseFlow, quote, quoteFlow);
 
-        BeraCrocLpErc20 lpToken = new BeraCrocLpErc20(base, quote, poolIdx);
+        bytes32 salt = keccak256(abi.encodePacked(base, quote));
+        BeraCrocLpErc20 lpToken = new BeraCrocLpErc20{salt: salt}(base, quote, poolIdx);
```

Comment:
Actually constructor arguments change the init code hash. In order to make the computation of LP address easier and deduplicate, we have to use Uniswap's pattern to init the pool.
https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol#L33

Reply:
applied changes